include/cucumber-cpp/internal/step/StepManager.hpp

```diff
@@ -62,6 +62,11 @@ class InvokeArgs {
 
     template<class T> T getInvokeArg(size_type i) const;
     const Table & getTableArg() const;
+
+    size_type argsSize() const {
+      return args.size();
+    }
+
```

Comment:
Sure.  I guess I've to rebase my stuff on master once all this and other fixes landed.  I've to review my code (which laid dormant for some time now), and will check what this function was required for.  In the context of the other changes this should be dropped therefore (I guess)